spec/rspec/rails/configuration_spec.rb

```diff
@@ -164,8 +164,14 @@ def in_inferring_type_from_location_environment
 
       group = RSpec.describe("Arbitrary Description", :use_fixtures)
 
-      expect(group).to respond_to(:fixture_path)
-      expect(group.fixture_path).to eq("custom/path")
+      if ::Rails::VERSION::MAJOR < 7
```

Comment:
I think `fixture_paths` were only introduced on the edge, as they are [missing in 7.0.4.3](https://github.com/rails/rails/blob/v7.0.4.3/activerecord/lib/active_record/test_fixtures.rb#L20). 
```suggestion
      if ::Rails::VERSION::STRING < '7.1.0'
```
Should work fine for `7.0.4.3`, `7.1.0-alpha` and `7.1.0`.